functional/OpenJcePlusTests/playlist.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TKG/playlist.xsd">
+	<test>
+		<testCaseName>openJcePlusTests</testCaseName>
```

Comment:
Can the framework run junit tests as individual tests such that they are reported with stack traces and as individual test passes and failures? I only noticed this looking at the results where after running 1800+ tests it reports just one test failed.

Reply:
In general, if the tests produce junit result report in xml, our framework will pick it up and show subtests. For example, [testReport link](https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder_JCK/2345/testReport/). However, it looks like junit itself does not provide the report. The OpenJcePlusTests do not produce any output files. It looks like we will need to leverage ant and junit Integration `<junit>` or maven for this.

It may be better to get PR in, so we can get the test running. Meanwhile, we will look into junit output for OpenJcePlusTests.